sample.env

```diff
@@ -209,4 +209,21 @@ RSU_STATUS_LOGGING_LEVEL="INFO"
 COUNTS_LOGGING_LEVEL="INFO"
 OBU_OTA_LOGGING_LEVEL="INFO"
 # Supported log levels are "ALL", "DEBUG", "ERROR", "FATAL", "INFO", "OFF", "TRACE" and "WARN"
-KC_LOGGING_LEVEL="INFO"
\ No newline at end of file
+KC_LOGGING_LEVEL="INFO"
+
+# Necessary Variables if using docker-compose-mongo.yml
+MONGO_ADMIN_DB_USER=root
+MONGO_ADMIN_DB_PASS=password
+MONGO_CV_MANAGER_DB_USER=root
+MONGO_CV_MANAGER_DB_PASS=password
+
+MONGO_IP=${DOCKER_HOST_IP}
+MONGO_DB_USER=root
+# This is the default password for the MongoDB container - MUST BE CHANGED FOR DEPLOYMENTS
+MONGO_DB_PASS=password
+MONGO_PORT=27017
+MONGO_DB_URI=mongodb://${MONGO_DB_USER}:${MONGO_DB_PASS}@${MONGO_IP}:${MONGO_PORT}/

```

Comment:
Moved these up to lines 53-54. Additionally, since there are two MONGO_DB_URI variables I've commented out the MONGO_DB_URI used by docker-compose-mongo.yml and made a note above it to uncomment if running MongoDB locally.